repo/packages/B/beta-mesosphere-jupyter-service/0/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "name": "beta-mesosphere-jupyter-service",
+  "version": "1.3.0-0.35.4-beta",
```

Comment:
remove `-beta` from version

Reply:
The `-beta` suffix aligned with how we've been versioning beta services for the SDK-based frameworks and clearly helps call out (again) that it's a service that's in Beta.